cms/static/sass/programs/_build.scss

```diff
@@ -0,0 +1,26 @@
+// ------------------------------
+// Programs: Main Style Compile
+
+// About: Sass compile for the Programs IDA.
+
+// #CONFIG:         app-centric configuration/overrides
+// #LIB:            third party libraries and dependencies
+// #EXTENSIONS:     add-ons and further customizations
+
+
+// ------------------------------
+// #CONFIG
+// ------------------------------
+$scope-class: 'program-app';
+// ------------------------------
+// #EXTENSIONS
+// ------------------------------
+.#{$scope-class} {
```

Comment:
I remember @AlasdairSwan describing how he had to implement scoping for styles in this app when it was embedded in Studio. Is this part of that, and should it be kept?